### PR TITLE
added policy props set/update

### PR DESCRIPTION
**Pull request by meirhalachmi**

### CODE REVIEW

File: src/corona_hakab_model/policies_manager.py [hunk 1]
```diff
@@ -78,15 +78,22 @@ def __init__(
             connection_change_factor: float,
             circle_conditions: Iterable[Callable[[Any], bool]],
             agent_conditions: Iterable[Callable[[Any], bool]] = None,
+            policy_props_update: Dict = None,
```
Comment (ghost) [inline src/corona_hakab_model/policies_manager.py:81]:
```suggestion
            policy_props_update: Dict = dict(),
```

File: src/corona_hakab_model/policies_manager.py [hunk 2]
```diff
@@ -78,15 +78,22 @@ def __init__(
...
         self.circle_conditions = circle_conditions
 
+        if policy_props_update is None:
+            policy_props_update = dict()
+
         if agent_conditions is None:
             agent_conditions = []
 
         self.agent_conditions = agent_conditions
+        self.policy_props_update = policy_props_update
```
Comment (ghost) [inline src/corona_hakab_model/policies_manager.py:95]: you have this initialization twice

File: src/corona_hakab_model/policies_manager.py (hunk 1 above)
Comment (meirhalachmi) [reply src/corona_hakab_model/policies_manager.py:81]: giving a param a default mutable value is problematic:
https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

File: src/corona_hakab_model/policies_manager.py (hunk 2 above)
Comment (meirhalachmi) [reply src/corona_hakab_model/policies_manager.py:95]: oops

File: src/corona_hakab_model/policies_manager.py (hunk 1 above)
Comment (ghost) [reply src/corona_hakab_model/policies_manager.py:81]: oh i didn't know about that, sounds good

**Review by ghost — APPROVED** [review]

**Pull request merged**